### PR TITLE
Load Testing and Documentation 

### PR DESCRIPTION
**Pull request by ghost**
# Load Testing Setup

- ADR generated for load testing
- Implemented [Locust.io](https://locust.io/) load testing
- added directory for load test config file

### CODE REVIEW

**Review by JohnNKing — COMMENTED** [review]
LGTM, but I'll defer to others for the approval

File: README.md [hunk 1]
```diff
@@ -63,6 +63,35 @@ This will start the API, wait for it to respond, run the end-to-end tests agains
...
+
+In a terminal, navigate to the directory *locustfile.py* is in and  run
+
+```shell
+locust
+```
+
+The terminal will start a local web interface and you can enter
+the swarm parameters for the test and the local url where the app is running
+(usually http//:localhost:8080)
```
Comment (halprin) [inline README.md:91]:
```suggestion
(usually http://localhost:8080).
```

File: README.md [hunk 2]
```diff
@@ -63,6 +63,35 @@ This will start the API, wait for it to respond, run the end-to-end tests agains
...
+```
+
+You can quickly validate the install with `locust -V`.
+
+A *locustfile.py* is required to run the tests and there is currently one located
+in `/operations`
+
+Run the application...
+
+In a terminal, navigate to the directory *locustfile.py* is in and  run
```
Comment (halprin) [inline README.md:83]:
```suggestion
In a terminal, navigate to the directory *locustfile.py* is in and run
```

File: README.md [hunk 3]
```diff
@@ -63,6 +63,35 @@ This will start the API, wait for it to respond, run the end-to-end tests agains
...
+Install Python (3.7 or later), then...
+
+```shell
+pip3 install locust
+```
+
+You can quickly validate the install with `locust -V`.
+
+A *locustfile.py* is required to run the tests and there is currently one located
+in `/operations`
```
Comment (halprin) [inline README.md:79]:
```suggestion
in `/operations`.
```

File: README.md [hunk 4]
```diff
@@ -63,6 +63,35 @@ This will start the API, wait for it to respond, run the end-to-end tests agains
...
+
+Run the application...
```
Comment (halprin) [inline README.md:81]: Is this a sub-heading or some sort?  Or is this supposed to be part of a sentence that goes with the above or below paragraphs?

**Review by halprin — CHANGES_REQUESTED** [review]
Up around line 23 of the `README.md`, I would add Python and Locust to the list additional development requirements.

File: README.md (hunk 4 above)
Comment (tjohnson7021) [reply README.md]: 'Run the application...' was a step. I think I fixed it in a way that makes more sense. Let me know what you think.

**Review by halprin — APPROVED** [review]
Thanks for making the changes!

**Pull request merged**